sim/druid/savage_roar.go

```diff
@@ -0,0 +1,54 @@
+package druid
+
+import (
+	"time"
+
+	"github.com/wowsims/wotlk/sim/core"
+	"github.com/wowsims/wotlk/sim/core/proto"
+	"github.com/wowsims/wotlk/sim/core/stats"
+)
+
+func (druid *Druid) getSavageRoarMultiplier() float64 {
+	glyphBonus := core.TernaryFloat64(druid.HasMajorGlyph(proto.DruidMajorGlyph_GlyphOfSavageRoar), 0.03, 0)
+	return 1.3 + glyphBonus
+}
+
+func (druid *Druid) registerSavageRoarSpell() {
+	actionID := core.ActionID{SpellID: 52610}
+	baseCost := 25.0
+
+	druid.SavageRoarAura = druid.RegisterAura(core.Aura{
+		Label:    "Savage Roar Aura",
+		ActionID: actionID,
+		Duration: 9,
+		OnGain: func(aura *core.Aura, sim *core.Simulation) {
+			druid.PseudoStats.PhysicalDamageDealtMultiplier *= druid.getSavageRoarMultiplier()

```

Comment:
Instead of calling this function every time, calculate it once outside the aura callbacks and capture the constant in the closure.